test/model/message_test.dart

```diff
@@ -168,32 +168,40 @@ void main() {
       await prepareOutboxMessage(destination: DmDestination(
         userIds: [eg.selfUser.userId, eg.otherUser.userId]));
       checkState().equals(OutboxMessageState.hidden);
+      checkNotNotified();
 
       async.elapse(kLocalEchoDebounceDuration);
       checkState().equals(OutboxMessageState.waiting);
+      checkNotifiedOnce();
 
       await receiveMessage(eg.dmMessage(from: eg.selfUser, to: [eg.otherUser]));
       check(store.outboxMessages).isEmpty();
+      checkNotifiedOnce();
```

Comment:
Maybe add all these checks as a prep commit? Then if there are any that change with the main change (or if there are none), that will be more visible.